### DISCUSSION TITLE
Cant exit emergency mode

### DISCUSSION THREAD

**Discussion by Dorfeuheinz** [linuxdeepin/developer-center#3897] [Troubleshooting | 故障排查]
Whenever I shut down my OS, even though its a safe shutdown, every app has been closed before shutdown, I still always enter into emergency mode upon restarting. & it doesnt allow me to type anything, all I can do is press enter, but nothing happens. 
Ive been stuck on this for months. Please help.
The problem persists till this day
![1ea2de1e-0132-4c56-ab80-a21db8f75df5](https://user-images.githubusercontent.com/79535302/225610850-1a53817a-5810-4bd3-bfa9-3c76909cd546.jpg)
Heres, the emergency mode, i cant type anything here. No other key other than enter gets registered. After pressing enter, the text only repeats itself

Kindly help me out, its been very troublesome

**Comment by Dorfeuheinz** [comment]
Could some one please help me out with this. Its getting a little frustrating having to reinstall the os everytime this happens

**Comment by BLumia** [comment]
Personally, I suggest you find a livecd, boot to the livecd, and try chroot to the system and see if you can repair them.

> Whenever I shut down my OS, even though its a safe shutdown, every app has been closed before shutdown, I still always enter into emergency mode upon restarting.

Does this happen every time you reboot the system? Even after a clean system re-installation and reboot the system immediately?

**Reply by Dorfeuheinz** [reply]
hey, thanks again for the help.

No, not immediately after re-installation. But theres no fixed pattern. If I shut down the system, it may or may not enter into emergency mode. Idk why does this happen. 
I did try everything, i honestly dont know why this is happening. Initially when this used to happen I just used to do fsck dev/sdaN & the prob. wud fix itself, but now it doesnt allow me to type anything

**Reply by Dorfeuheinz** [reply]
I even did a thorough check for badblocks in the HDD in the past, but everything was good. So idk why is this issue showing up now

**Comment by myml** [comment]
Can I enter the deepin system if restart again?

if you have live os, can check for any errors in the /var/log/boot.log

**Reply by Dorfeuheinz** [reply]
live os as in the usb with the iso image right? I have that yes, now that ive re-installed the os 20-30 times.

Im sorry, I couldnt understand your 1st question sir. Could you elaborate please?

**Reply by myml** [reply]
Try the:
1. Boot the live system from the USB
2. Press `CTRL+F2` swtich to tty2
4. Mount computer hard drives
5. Review the boot log

**Reply by Dorfeuheinz** [reply]
Ok, ill try that, the next time this happens. Though what am I looking for? As in, am I supposed to fsck the system disk partition or something? Anything that I might have to edit?

**Comment by Dorfeuheinz** [comment]
My boot.log file is empty for some reason. Idk why
Heres the journalctl_log.txt file-
[journalctl_log.txt](https://github.com/linuxdeepin/developer-center/files/11150002/journalctl_log.txt)

Kindly lemme know what to do next

**Reply by myml** [reply]
> My boot.log file is empty for some reason. Idk why Heres the journalctl_log.txt file- [journalctl_log.txt](https://github.com/linuxdeepin/developer-center/files/11150002/journalctl_log.txt)
> 
> Kindly lemme know what to do next

Sorry, I can not find problem from this log file.

**Reply by myml** [reply]
Is this file copy from a system in emergency mode?

**Reply by raspin0** [reply]
Yes, we got a report of this issue on Discord as well and tried to collect logs for you to investigate. 
Commands we've used:
```
cd ~
mkdir logs && cd logs
journalctl -xb --since "2023-04-04 00:00:00" > journalctl_log.txt
sudo cp /var/log/boot.log boot.log
```
The author noted that the `boot.log` file was **empty** even when trying to access it directly through `sudo nano /var/log/boot.log`
It's also worth mentioning that the deepin OS has been reinstalled multiple times at this point with issue reoccurring every time

**Reply by Dorfeuheinz** [reply]
That us correct. Idk what seems to be the issue really, & why is the boot.log empty & what does it even mean. How is this even happening?

**Reply by myml** [reply]
journalctl command or '/var/log' are both logs of the current system.

You should view the logs of system that have emergency mode,not those that are currently in use use and has no problems. 

This requires you to hava another os, either another computer or live system boot from usb(like Puppy Linux).
